src/posture_estimator.cpp

```diff
@@ -160,16 +163,122 @@ bool PostureEstimator::updateCurrentPoseAndCheckPosture(
   return this->good_posture;
 }
 
+void PostureEstimator::display_current_pose(
+    PostureEstimating::Pose current_pose, cv::Mat current_frame,
+    bool ideal_pose_set) {
+  int imageWidth = current_frame.cols;
+  int imageHeight = current_frame.rows;
+
+  for (int i = JointMin + 1; i <= JointMax - 2; i++) {
+    // Only consider the Head, Neck, Shoulder and Hip joints
```

Comment:
Any reason why we aren't including knees and feet? Is it just we are expecting users not to show their legs on camera? Should we add a user setting that sets what body parts are considered?

Reply:
Personally I am unable to get into a position where my legs are also visible by the camera and when we discussed in a previous PR I thought we agreed this would be okay? We can have a setting yes, but thought we should do this first as there isn't much time left

Reply:
I agree. It's also important to consider usability at this point and if *we* are struggling to set the thing up with the whole body in view (and useable) then I expect a new user will not manage it and get confused why the output is jumbled. We could hide it in settings as an option, but I think it is a low priority and niche use-case so I don't think we should aim to support that right now.

Reply:
That's fair